modules/artifact/src/main/java/org/mule/module/artifact/classloader/DefaultResourceReleaser.java

```diff
@@ -188,17 +195,60 @@ private boolean hasDeclaredMethod(Class<?> clazz, String methodName, Class<?>...
    */
   private void shutdownMySqlAbandonedConnectionCleanupThread() {
 
-    Class<?> classAbandonedConnectionCleanupThread;
+    Class<?> classAbandonedConnectionCleanupThread = null;
     try {
-      classAbandonedConnectionCleanupThread =
-          this.getClass().getClassLoader().loadClass("com.mysql.jdbc.AbandonedConnectionCleanupThread");
+      for (String knownCleanupThreadClassAddress : CONNECTION_CLEANUP_THREAD_KNOWN_CLASS_ADDRESES) {
+        try {
+          classAbandonedConnectionCleanupThread =
+              this.getClass().getClassLoader().loadClass(knownCleanupThreadClassAddress);
+        } catch (ClassNotFoundException e) {
+          logger.warn("No AbandonedConnectionCleanupThread registered with class address " + knownCleanupThreadClassAddress);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Format specifiers should be used instead of string concatenation. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7685&issues=AWjzLS16otIXnmsUYkAu&open=AWjzLS16otIXnmsUYkAu)

[ISSUE_KEY]: AWjzLS16otIXnmsUYkAu